rules/context.go

```diff
@@ -3,7 +3,8 @@ package rules
 import "time"
 
 type Context struct {
-	Text string
+	Source string
```

Comment:
Why do we need this here? I thought that it's better to capture surrounding text if needed by the rule, isn't it?